sdk/azcore/messaging/cloud_event.go

```diff
@@ -0,0 +1,291 @@
+// Copyright 2017 Microsoft Corporation. All rights reserved.
+// Use of this source code is governed by an MIT
+// license that can be found in the LICENSE file.
```

Comment:
This copyright is non-standard with the year.

Reply:
Good catch.  I'll fix this in `main` then cherry-pick the commit.

Reply:
https://github.com/Azure/azure-sdk-for-go/pull/21695

Reply:
Fixed.